cnf-certification-test/networking/suite.go

```diff
@@ -179,3 +184,51 @@ func testNetworkConnectivity(env *provider.TestEnvironment, aIPVersion netcommon
 		ginkgo.Fail(fmt.Sprintf("%d nets failed the %s network %s ping test.", n, aType, aIPVersion))
 	}
 }
+
+func testOCPReservedPortsUsage(env *provider.TestEnvironment) {
+	// List of all ports reserved by Openshift
+	OCPReservedPorts := map[int32]bool{22623: true, 22624: true}
+
+	var failedContainers int
+	var rogueContainers []string
+	for _, cut := range env.Containers {
+		// First check if any of the containers under test has declared a port reserved by OCP
+		for _, port := range cut.Data.Ports {
+			if OCPReservedPorts[port.ContainerPort] {
+				tnf.ClaimFilePrintf("%s has declared a port (%d) reserved by Openshift", cut.String(), port.ContainerPort)
```

Comment:
Just a reminder: there's no need to explicitly use the `.String()` methods when printing our "cut" or "put" objects, as that stringer method will be called by printer functions like printf, logrus.Infof (or this "ClaimFilePrintf") when they're used with %s or %v formatters.

Reply:
Done.